src/scss/_header.scss

```diff
@@ -33,4 +33,18 @@
 		height: 1px;
 	}
 
+	@each $breakpoint in ('M', 'L', 'XL') {
+		.o-header__show--#{$breakpoint} {
```

Comment:
Or perhaps we could offer a more descriptive interface, one more similar to o-grid, E.G.:

```html
<div class="o-header__nav" data-o-header-display="hide Lshow">
<div class="o-header__nav" data-o-header-display="show Mhide">
```